### PR TITLE
Add AWS Full Size Tests

### PR DESCRIPTION
**Pull request by apeltzer**
Started working on full-test profile, needs proper human reference + gtf now though

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/scrnaseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/scrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_full.config [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
-
-    // Genome references
-    genome = 'R64-1-1'
+    // Input data
+    input        = 'https://raw.githubusercontent.com/nf-core/test-datasets/scrnaseq/samplesheet_2.0_full.csv'
+    genome_fasta = 's3://nf-core-awsmegatests/scrnaseq/input_data/Homo_sapiens.GRCh38.dna.primary_assembly.fa.modified'
+    gtf          = 's3://nf-core-awsmegatests/scrnaseq/input_data/gencode.v32.primary_assembly.annotation.gtf.filtered'
+    protocol     = '10XV2'
+    // Ignore `--input` as otherwise the parameter validation will throw an error
+    schema_ignore_params = 'genomes,input_paths,input'
```
Comment (ggabernet) [inline conf/test_full.config:23]:
```suggestion
    schema_ignore_params = 'genomes'
```

Comment (ggabernet) [reply conf/test_full.config:23]: I think this is not needed, as `input` is provided

**Review by ggabernet — APPROVED** [review]
Just got a comment but looks good to me otherwise

File: conf/test_full.config [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
     Use as follows:
-        nextflow run nf-core/scrnaseq -profile test_full,<docker/singularity> --outdir <OUTDIR>
+        nextflow run nf-core/scrnaseq -profile test,<docker/singularity> --outdir <OUTDIR>
 
 ----------------------------------------------------------------------------------------
 */
 
 params {
-    config_profile_name        = 'Full test profile'
+    config_profile_name        = 'Test profile'
```
[the remark this answers is not in the document]
Comment (fmalmeida) [reply conf/test_full.config:14]: I thought the same. Otherwise everything looks good.

**Review by fmalmeida — APPROVED** [review]

**Pull request merged**